lib/VMwareWebService/MiqVimVm.rb

```diff
@@ -751,7 +751,7 @@ def addDisk(backingFile, sizeInMB, label=nil, summary=nil)
 							# bck.diskMode = VirtualDiskMode::Independent_nonpersistent
 							bck.diskMode		= VirtualDiskMode::Independent_persistent
 						    bck.split			= "false"
-						    bck.thinProvisioned	= "false"
+						    bck.thinProvisioned	= thinProvisioned.to_s
```

Comment:
My idea was the same - people who used to work with raw vCenter API quite often pass everything as strings. (Ok, at least it is true for our team.)